lib/loaders/eslint.js

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of the Symfony Webpack Encore package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+const loaderFeatures = require('../features');
+
+/**
+ * @param {WebpackConfig} webpackConfig
+ * @return {Object} of options to use for eslint-loader options.
+ */
+module.exports = {
+    getOptions(webpackConfig) {
+        loaderFeatures.ensurePackagesExist('eslint');
+
+        const eslintLoaderOptions = {
+            parser: 'babel-eslint',
+            emitWarning: true,
+            rules: {
+                'linebreak-style': 'off'
```

Comment:
Should we really disable that rule by default?

Reply:
Given most people on windows have git doing a "checkout crlf, commit lf", I feel it's an option that should be default everywhere.

Reply:
I'm not sure that should be Git's job to do that kind of thing, especially since that's not its default behavior on Windows (you have to switch the `core.autocrlf` manually).

A cleaner way to handle that IMO is to use an `.editorconfig` file (which is supported by many editors) or change manually the config of your IDE so it doesn't use CRLF line breaks.

Note that this rule is also enabled in Airbnb and Google ESLint presets:

https://github.com/airbnb/javascript/blob/69a3554100935b79b9c444102504f98103834079/packages/eslint-config-airbnb-base/rules/style.js#L158-L160

https://github.com/google/eslint-config-google/blob/394bf3c9f858b83514fdcbf23d74492e253d611f/index.js#L214

Reply:
Hmm, true. With default, I was referring to tools like the github desktop client and Atlassian sourcetree. I'll remove this config :)